Sources/Components/ARCProgressBar.swift

```diff
@@ -81,15 +81,30 @@ struct CapsuleView: View {
         return 1 / (1 - (1 - leftOver))
     }
 
+    /// Calculate the top capsule width
+    var progressWidth: CGFloat {
+        let progress = isPartiallyFilled ? capsuleWidth / CGFloat(partiallyDividedBy) : capsuleWidth
+        return progress + (.ArcProgressBar.capsuleTrailingWidth / 2) // For perfect centering
+    }
+
     var body: some View {
         ZStack(alignment: .leading) {
             Capsule().fill(emptyColor)
             if isFilledCapsule {
-                Capsule().fill(filledColor)
-                    .frame(
-                        width: isPartiallyFilled ? capsuleWidth / CGFloat(partiallyDividedBy) : capsuleWidth,
-                        height: capsuleHeight
-                    )
+                ZStack {
+                    Capsule().fill(filledColor)
+                        .frame(
+                            width: progressWidth,
+                            height: capsuleHeight
+                        )
+                    if isPartiallyFilled {
+                        Rectangle()
+                            .fill(Color.arcWhite)
+                            .frame(maxWidth: .ArcProgressBar.capsuleTrailingWidth, alignment: .trailing)
+                            .frame(maxWidth: .infinity, alignment: .trailing)
```

Comment:
Do we need two `.frame(maxWidth:)`?

Reply:
@BenShutt So I've discovered something super cool.
```
VStack {
content
.frame(maxWidth: 10)
}
.frame(maxWidth: .infinity)
```
is the same with
```
content
.frame(maxWidth: 10)
.frame(maxWidth: .infinity)
```
You gotta love the legos 😄